pro-planner-client/src/components/UserList.jsx

```diff
@@ -1,70 +1,154 @@
+	};
+
+	const handleContinueClick = () => {
+		if (selectedUser) {
+			dispatch(resetError());
+			navigate(`/${tripId}`);
+		} else {
+			dispatch(
+				setError({
+					errType: ERR_TYPE.ERR,
+					message: 'Please select a user to continue.',
+				})
+			);
+			return;
+		}
+	};
+
+	const toggleEdit = () => {
+		if (isEditing) {
+			setIsEditing(false);
+		} else {
+			setIsEditing(true);
+		}
+	};
 
-    if (!userList || userList.length === 0) {
-        return <p>No User Yet, Please add a user.</p>
-    }
+	const handleDelete = userToDelete => {
+		if (usersToDelete.includes(userToDelete)) {
+			// If the userToDelete is already in the usersToDelete list, remove it
+			const updatedUsersToDelete = usersToDelete.filter(
+				user => user !== userToDelete
+			);
+			setUsersToDelete(updatedUsersToDelete);
+		} else {
+			// If the userToDelete is not in the usersToDelete list, add it
+			setUsersToDelete([...usersToDelete, userToDelete]);
+		}
+	};
 
-    const handleUserSelectChange = (user) => {
-        dispatch(selectUser(user));
-    };
+	const handleCancel = () => {
+		setIsEditing(false);
+		setUsersToDelete([]);
+	};
 
-    const handleContinueClick = () => {
-        if (selectedUser) {
-            dispatch(resetError());
-            navigate(`/${tripId}`);
-        } else {
-            dispatch(setError({
-                errType: ERR_TYPE.ERR,
-                message: 'Please select a user to continue.',
-            }));
-            return;
-        }
-    };
+	const handleConfirm = () => {
+		if (usersToDelete.length !== 0) {
+			dispatch(deleteUsersAsync({ planId: tripId, usersToDelete }));
+		}
+		setUsersToDelete([]);
+		setIsEditing(false);
+	};
 
-    return (
-        <>
-            <Form>
-                <Form.Label>Which one is you?</Form.Label>
-                <div style={{width: '100%'}}>
-                    {userList.map((user) => (
-                        <div key={user}>
-                            <ToggleButton
-                                style={{width: '100%'}}
-                                className="mb-2"
-                                variant="outline-secondary"
-                                type="checkbox"
-                                id={user}
-                                checked={selectedUser === user}
-                                onChange={() => handleUserSelectChange(user)}
-                                value={user}
-                            >
-                                {user}
-                            </ToggleButton>
-                        </div>
-                    ))}
-                </div>
-                <Button style={{width: '100%'}}
-                        onClick={handleContinueClick}>
-                    Continue
-                </Button>
-            </Form>
-        </>
-    );
-}
+	return (
+		<>
+			<Form>
+				<div style={{ width: '100%' }}>
+					<Row>
+						<Col>
+							<Form.Label>Which one is you?</Form.Label>
```

Comment:
May be worth changing this string when in edit mode to instruct the user on what they can do.